src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/history/repositories/AccessionHistoryRepository.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.commons.accession.persistence.history.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.NoRepositoryBean;
+import uk.ac.ebi.ampt2d.commons.accession.core.AccessionStatus;
+
+import java.io.Serializable;
+import java.util.Collection;
+
+@NoRepositoryBean
+public interface AccessionHistoryRepository<ENTITY, ID extends Serializable>
+        extends CrudRepository<ENTITY, ID> {
+
+    Collection<ENTITY> findAllByAccessionStatus(AccessionStatus accessionStatus);
```

Comment:
According to [this post and documentation](https://stackoverflow.com/a/37321365), this would be equivalent to `findByAccessionStatus`. That sounds less redundant and also follows the same naming style that we already use in variation-commons.